### PR TITLE
Fix case with variance eq to 0

### PR DESCRIPTION
**Pull request by amallia**

### CODE REVIEW

File: include/taily.hpp [hunk 1]
```diff
@@ -137,10 +138,10 @@ struct Query_Statistics {
 [[nodiscard]] auto
 fit_distribution(Feature_Statistics const& query_term_stats) -> boost::math::gamma_distribution<>
 {
-    const double k = std::pow(query_term_stats.expected_value, 2.0)
-        / query_term_stats.variance;
-    const double theta =
-        query_term_stats.variance / query_term_stats.expected_value;
+    double epsilon = std::numeric_limits<double>::epsilon;
```
Comment (elshize) [inline include/taily.hpp]: `epsilon` is a member function of `numeric_limits`, you need to add `()`.

Comment (amallia) [reply include/taily.hpp]: Sorry, a typo while copy-pasting

Comment (amallia) [reply include/taily.hpp]: Btw, do you think it makes sense logically?

Comment (elshize) [reply include/taily.hpp]: It's certainly better than it was, so I think we can merge it. Let me think about this though, and see if we can have a more exact corner-case. But I need to look at the paper first to refresh my memory.

**Pull request merged**